wordpress/wp-content/plugins/cds-wpml-mods/resources/js/util/fetch.js

```diff
@@ -1,11 +1,14 @@
-const REST_URL = 'http://localhost/wp-json/';
+const CDS_VARS = window.CDS_VARS || {
+    rest_url: 'http://localhost/wp-json/',
+    local: true
+};
 
 export const getData = async (endpoint) => {
     const requestHeaders = new Headers();
     // do we need this?
     //requestHeaders.append('X-WP-Nonce', CDS_VARS.rest_nonce);
```

Comment:
Used on the auth end of things

See: https://developer.wordpress.org/rest-api/using-the-rest-api/authentication/#cookie-authentication